package/kedro_viz/launchers/cli.py

```diff
@@ -129,6 +131,22 @@ def run(
     """Launch local Kedro Viz instance"""
     from kedro_viz.server import run_server
 
+    kedro_project_path = _find_kedro_project(Path.cwd())
+
+    if kedro_project_path is None:
+        display_cli_message(
+            "ERROR: Failed to start Kedro-Viz : "
+            "Could not find the project configuration "
+            f"file '{_PYPROJECT}' in '{Path.cwd()}'. "
+            f"If you have created your project with Kedro "
+            f"version <0.17.0, make sure to update your project template. "
+            f"See https://github.com/kedro-org/kedro/blob/main/RELEASE.md"
+            f"#migration-guide-from-kedro-016-to-kedro-0170 "
+            f"for how to migrate your Kedro project.",
+            "red",
```

Comment:
The link is not correct ^ https://github.com/kedro-org/kedro/blob/main/RELEASE.md#migration-guide-from-kedro-016-to-017

Reply:
Maybe we don't have this because kedro-viz does not support Kedro 17 or 16.

Reply:
Yeah, could just say that Kedro viz is looking for a `pyproject.toml` file

Reply:
I was trying to mimick what kedro had. But yes @rashidakanchwala since viz does not suppport these versions, can I just have the message as below ?

```
"ERROR: Failed to start Kedro-Viz : "
"Could not find the project configuration "
f"file '{_PYPROJECT}' in '{Path.cwd()}'. "
```